packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -49,6 +49,12 @@ contract L1Block is ISemver {
     /// @notice The latest L1 blob base fee.
     uint256 public blobBaseFee;
 
+    /// @notice The size of the interop dependency set.
+    uint8 public interopSetSize;
+
+    /// @notice The chain IDs in the interop dependency set.
+    uint256[] public chainIds;
```

Comment:
can we rename this to be `dependencySet` to match the spec?

Reply:
see https://github.com/ethereum-optimism/optimism/pull/9690#discussion_r1516722809